Move math::Axpy function to elementwise lib

xiaomengy opened the pull request:
Summary: Move math::Axpy function to elementwise lib

Differential Revision: D14574697

BIT-silence commented:
Benchmark:

WeightedSumOp for two Inputs with size [256, 256, 112, 112] and in-place for first input,
53.1ms -> 42.9ms

houseroad reviewed (approved):
Looks good.

xiaomengy committed:
Move math::Axpy function to elementwise lib (#18316)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/18316

Move math::Axpy function to elementwise lib

i-am-not-moving-c2-to-c10

Reviewed By: houseroad

Differential Revision: D14574697

fbshipit-source-id: 45283e9122fc0fb5dc87c7170fc7b9f08e3cbf58
